generated/nidcpower/nidcpower/_grpc_stub_interpreter.py

```diff
@@ -25,9 +25,12 @@ def __init__(self, grpc_options):
         self._client = nidcpower_grpc.NiDCPowerStub(grpc_options.grpc_channel)
         self._vi = 0
 
-    def _invoke(self, func, request):
+    def _invoke(self, func, request, metadata=None):
         try:
-            response = func(request)
+            if metadata is not None:
+                response = func(request, metadata=metadata)
+            else:
+                response = func(request)
```

Comment:
If `func` is the client stub, I don't think this if-statement is necessary. It accepts `metadata=None`.
https://grpc.github.io/grpc/python/grpc.html#multi-callable-interfaces

Reply:
For python itself, it's not necessary, but passing along metadata=None would require us to update all the assert_called_with invocations in the tests.